runtime/frequency/src/apis.rs

```diff
@@ -149,11 +149,32 @@ impl_runtime_apis! {
 		}
 
 		fn get_preset(id: &Option<sp_genesis_builder::PresetId>) -> Option<Vec<u8>> {
-			get_preset::<RuntimeGenesisConfig>(id, |_| None)
+			get_preset::<RuntimeGenesisConfig>(id,  &crate::genesis::presets::get_preset)
 		}
 
 		fn preset_names() -> Vec<sp_genesis_builder::PresetId> {
-			Default::default()
+			let mut presets = vec![];
+
+			#[cfg(any(
+				feature = "frequency-no-relay",
+				feature = "frequency-local",
+				feature = "frequency-lint-check"
+			))]
+			presets.extend(
+			vec![
+				sp_genesis_builder::PresetId::from("development"),
+				sp_genesis_builder::PresetId::from("frequency-local"),
+				sp_genesis_builder::PresetId::from("frequency"),
```

Comment:
question: why do you need `frequency` for local features?

Reply:
Good question! This is something I mentioned during our stand-up, and I believe we agreed that having the frequency feature enabled by default was ok.